Cargo.toml

```diff
@@ -11,16 +11,20 @@ name = "pgrx_embed_pg_no_seqscan"
 path = "./src/bin/pgrx_embed.rs"
 
 [features]
-default = ["pg15"]
+default = ["pg17"]
```

Comment:
Use the latest version as default, as this is the one which is closest to the future version.